code/datums/components/jetpack.dm

```diff
@@ -92,14 +92,14 @@
 
 /datum/component/jetpack/proc/deactivate(datum/source)
 	SIGNAL_HANDLER
-	QDEL_NULL(trail)
 	var/mob/moving = get_mover.Invoke()
 	if(moving)
 		UnregisterSignal(moving, COMSIG_MOVABLE_MOVED)
 		UnregisterSignal(moving, COMSIG_MOVABLE_PRE_MOVE)
 		UnregisterSignal(moving, COMSIG_MOVABLE_SPACEMOVE)
 		UnregisterSignal(moving, COMSIG_MOVABLE_DRIFT_VISUAL_ATTEMPT)
 		UnregisterSignal(moving, COMSIG_MOVABLE_DRIFT_BLOCK_INPUT)
```

Comment:
this shouldn't be done in this PR but someone should fix this now that `UnregisterSignal()` accepts lists.